sim_pipeline/gg_lens_table.py

```diff
@@ -0,0 +1,39 @@
+from astropy.table import Table
+from sim_pipeline.Pipelines.skypy_pipeline import SkyPyPipeline
+from sim_pipeline.gg_lens import GGLens, theta_e_when_source_infinity
+
+import numpy as np
+
+def create_table(gg_lens_population):
+    """
+    Creates an astropy table for a given gg_lens population.
+
+    :param gg_lens_population: List of GGLens instances.
+    :type gg_lens_population: list
+    :return: Astropy table with lens properties.
+    :rtype: `~astropy.table.Table`
+    """
+
+    # get the various properties
+    einstein_radii = [lens.get_einstein_radius() for lens in gg_lens_population]

```

Comment:
looping through the gg_lens_pop class for each individual variable to extract might not be the fastest. Can you make one for-loop and then read out all the variables at once? Might be worth checking which approach is faster.